.github/workflows/linux.yml

```
@@ -111,6 +111,7 @@ jobs:
       run: |
         sudo apt-get install ccache
         ccache --max-size=2G
+        mkdir ${HOME}/.ccache
```

This will die if the cache _did_ exist.

```suggestion
        mkdir -p ${HOME}/.ccache
```